modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/PythonClientCodegen.java

```diff
@@ -260,6 +260,13 @@ public String toModelImport(String name) {
         return modelImport;
     }
 
+    @Override
+    protected void addImport(CodegenModel m, String type) {
+        if (m.parent != null && m.parent.equals(type)) {
+            super.addImport(m, type);
+        }
+    }
+
```

Comment:
Only adds the parent class for importing as the python client does not need to know the type of the values it works with, but does need to import the parent class